src/cookie.js

```diff
@@ -42,17 +42,26 @@ async function main() {
     let {host} = new URL(tab.url);
     let domain = document.querySelector("#domain");
     domain.textContent = host;
-
+    let report = document.getElementById("report");
     let site = await getSite(host);
-    if (site) {
-        document.body.className = "managed";
+
+    if (tab.url.startsWith("about:") || tab.url.startsWith("file:") || tab.url == "") {
+        console.log("Condition met");
+        report.innerHTML = "This site is not a webpage...";
```

Comment:
A good rule of thumb is "never use `.innerHTML`".  This can be `.textContent` instead, but let's try to keep any text or styles out of the javascript file (which should be just for logic).

I would add a separate "This is not a web page" label, and control its visibility via a third class for body perhaps "invalid", to go along with "managed" and "unknown".

Reply:
Good to know.
Label is added and adjusted for.